doc/replication.md

```diff
@@ -305,8 +290,9 @@ Replicator replicator = ReplicatorFactory.oneway(pullReplication);
 The following APIs are still supported but deprecated. They will be soon removed from the library.
```

Comment:
Done in 69e194a, I'll stick a post-it on my screen / do something to remember to update it